proposals/akri-resources-status.md

```diff
@@ -0,0 +1,78 @@
+# Status of akri resources
+
+Currently, both akri managed resources (`Instances` and `Configurations`) only have a `spec` and `metadata` field, this doesn't allow the akri
+components to give feedback to the user. In order to do so, we can add a `status` field to the resources, this document describes how this field
+should be populated.
+
+## Instances
+
+For an instance, the status should give insight about the broker resources (if any), to do that, you can use the following conditions:
+
+- Healthy: The instance is currently detected by an agent, "True" (instance detected) and "Unknown" (in grace period) are the only valid statuses
+- BrokerScheduled: The broker resources are all created (absent if no broker)
+- BrokerReady: All broker resources are "Ready" or "Succeeded"
+- Ready: All above conditions are True
```

Comment:
What if no brokers are deployed for an instance? what does `BrokerReady` display as?

Reply:
If there are no defined brokers to be deployed for an Instance, `BrokerReady` condition would simply not be present and `Ready` would simple reflect the `Healthy` condition, similar to how `BrokerScheduled` behave. 
I'll update the proposal to add the same comment that exists for `BrokerScheduled`.